src/styles/bloks/_calendar.scss

```
@@ -0,0 +1,28 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  max-width: $calendar-size;
+  padding: 20px 0;
+  margin: 0 auto;
+  &__cell {
+    @include square($cell-size);
+    position: relative;
+    background-color: $cell-color;
+    border: 1px solid black;
+    transition: background-color 0.5s, transform 0.5s;
```

Create variables for transition duration
```suggestion
    transition: background-color $transitionDuration, transform $transitionDuration;
```